credits.html

```diff
@@ -20,114 +20,212 @@
-          <span class="sr-only">Toggle navigation</span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-        </button>     
-      </div>
-      <div class="collapse navbar-collapse" id="bs">
-        <div class="nav-wrapper">
-          <div class="languagepicker roundborders large">
-            <a class="lang" data-href="language">Select your language</a>
-            <a class="translate translate-language-choice" id="en">
-			  <img src="icons/UnitedKingdom.png" alt="" class="translation-flag" /> English
-            </a>
-            <a class="translate translate-language-choice" id="si">
-              <img src="icons/SriLanka.png" alt="" class="translation-flag" /> Sinhala
-            </a>
-		  </div>
-            <ul class="nav navbar-nav navbar-right">
-              <li><a href="index.html" class="lang" data-href="game">GAME</a></li>
-              <li><a href="howtoplay.html" class="lang" data-href="how">HOW TO PLAY</a></li>
-              <li><a href="https://github.com/fossasia/labyrinth" class="lang" data-href="how">CONTRIBUTE</a></li>
-            </ul>
-            <img class="brand-logo center" alt="FOSSASIA" src="images/logos/labyrinthlogofull.png" width="80" height="60">
+  <div class="navbar navbar-default">
+  <div class="container-fluid">
+ 
+        <button class="navbar-toggle" data-toggle="collapse" data-target="#menubar">
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span> 
+          </button> 
+        <div class="navbar-header" >
+            <h1>Labyrinth</h1>
         </div>
-      </div>
-    </div>
-  </nav>
-    
 
+        <ul class="nav navbar-nav navbar-right collapse navbar-collapse " id="menubar">
+          <li>
+            <a href="howtoplay.html" class="lang " data-href="how"><span>How to Play</span></a>
+          </li>
+          <li>
+            <a href="index.html" class="lang " data-href="game"><span>Game</span></a>
+          </li>
+          <li>
+              <a href="https://github.com/fossasia/labyrinth" class="lang " ><span>Contribute</span></a>
+          </li>
+          <li class="select-your-lang" id="select-lang">
+            <a class="lang dropdown-toggle" data-href="language" id="dropdownMenu" data-toggle="dropdown" data-hover="dropdown">Select your language <span class="fa fa-caret-down"></span></a>
+            <ul class="dropdown-menu" aria-labelledby="dropdownMenu">
+            <li><a class="translate translate-language-choice" id="en">
+              <img src="icons/UnitedKingdom.png" alt="" class="translation-flag" /> English
+            </a></li>
+            <li><a class="translate translate-language-choice" id="si">
+              <img src="icons/SriLanka.png" alt="" class="translation-flag" /> Sinhala
+            </a></li>
+          </ul>
+          </li>
+        </ul>
+  </div>
```

Comment:
Hi, there is some indentation mismatch. No big deal. If the tests run through, this also means, we have a valid HTML page.

Reply:
@niccokunzmann I guess its only for a part of the page. I might have forgotten to indent that.
But I think that isn't an issue.